addon/services/flash-messages-service.js

```diff
@@ -12,14 +13,7 @@ const {
 export default Ember.Service.extend({
   queue               : emberArray([]),
   isEmpty             : computed.equal('queue.length', 0),
-
-  // refactor
-  defaultTimeout      : 3000,
-  defaultPriority     : 100,
-  defaultSticky       : false,
-  defaultShowProgress : false,
   defaultTypes        : [ 'success', 'info', 'warning', 'danger', 'alert', 'secondary' ],
```

Comment:
We could probably move defaultTypes into the global config as well. That way you could overwrite the thing if you didn't need any of those types. It would then make registering types manually with registerType redundant and we can make that a private method